### PR TITLE
Add hover effect

### LINKED ISSUE #98: Update Chips for hover / active effects

### PR DESCRIPTION
**Pull request by ChuckBTaylor**
Vika's added gover effects to the Chips. Here ya go.

Closes #98

### CODE REVIEW

File: src/assets/pages/directorProducer.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { IVideoListPageInfo } from '../../common/interfaces/IVideoListPageInfo'
 export const directorProducerPageInfo: IVideoListPageInfo = {
   pageTitle: 'Director & Producer',
   videoTitles: [
-    'The Indicators',
+    // 'The Indicators',
```
Comment (ChuckBTaylor) [inline src/assets/pages/directorProducer.ts:6]: Kurt said to leave this our for now

File: src/components/atoms/chipsForEnum/ChipsForEnum.tsx [hunk 2]
```diff
@@ -1,23 +1,36 @@
...
+  const theme = useTheme()
+  const buttonColor = theme.palette[value].main
 
   const handleClick = () => {
     navigate(url)
   }
 
   return (
     <Chip
+      component={Link}
```
Comment (ChuckBTaylor) [inline src/components/atoms/chipsForEnum/ChipsForEnum.tsx:18]: Swtich to using explicit `Link` component

File: src/components/atoms/chipsForEnum/ChipsForEnum.tsx [hunk 3]
```diff
@@ -1,23 +1,36 @@
...
+      to={url}
       label={label.toLocaleUpperCase()}
       color={value}
-      component="button"
       variant="outlined"
       onClick={handleClick}
+      sx={{
+        '&&:hover': {
```
Comment (ChuckBTaylor) [inline src/components/atoms/chipsForEnum/ChipsForEnum.tsx:25]: Stole this from here: [stack overflow](https://stackoverflow.com/questions/63349154/change-material-ui-outlined-chip-focus-and-hover-color)

File: src/components/atoms/chipsForEnum/ChipsForEnum.tsx (hunk 2 above)
Comment (mwilkins74) [reply src/components/atoms/chipsForEnum/ChipsForEnum.tsx:18]: Very cool

**Review by mwilkins74 — APPROVED** [review]
Love it!

**Pull request merged**